node/cmd/ccq/query_server.go

```diff
@@ -181,6 +187,41 @@ func runQueryServer(cmd *cobra.Command, args []string) {
 			}
 		}()
 	}
+	// Start prometheus scraping
```

Comment:
Looks good, but could you please move this into a separate function in status.go and call it at line 182 (only if statusAddr is set)?

Reply:
Done